src/2a_analyze_MPA.jl

```diff
@@ -36,6 +36,60 @@ include("common.jl")
         return nothing
     end
 
+        """
+        filter_distances(
+            target_rast::Raster,
+            gdf::DataFrame,
+            dist_nm
+        )::Raster
+
+    Filter pixels from target_raster that are within `dist_nm` (nautical miles) from a geometry
```

Comment:
"Filter pixels ... that are within `dist_nm` "

Is this correct? It should like we're removing data that is within a certain distance